### DISCUSSION TITLE
(Spanish) ya compile, pero no veo el ejecutable

### DISCUSSION THREAD

**Discussion by eggbanana173** [nesbox/TIC-80#1641] [Q&A]
hola, ya me informe un poco, lo configure bien y lo compile pero, no veo el ejecutable, ¿Qué hago?

PD: No vengo a molestarlo, solo vengo a preguntar

**Comment by nesbox** [comment]
pls go to `build/bin` directory

**Reply by chriztheanvill** [reply]
After the compile, could just move the bin file (tic80) to another folder and delete all git clone and build folders?

**Reply by nesbox** [reply]
Unfortunately, it's not an easy task because a lot of build scripts depends of this folder, I recommend you to write your own script with moving `tic80` and cleaning and call it whenever you want.

**Reply by chriztheanvill** [reply]
I think I figured out, in my case, using linux:
```sh
sudo apt-get install g++ git cmake ruby-full libglvnd-dev libglu1-mesa-dev freeglut3-dev libasound2-dev -y
git clone --recursive https://github.com/nesbox/TIC-80 && cd TIC-80/build
cmake ..

```
If I use `make -j4` will build it in the same folder, then should move this build to the correct (personal choise) folder: /usr/local/bin or ~/.local/bin
But If I use `sudo make install -j4` will place it correctly: /usr/local/bin

Am I right?

**Reply by nesbox** [reply]
I think yes, pls try `make install` and let us know :)
Thanks.

**Reply by eggbanana173** [reply]
> pls go to `build/bin` directory

perdón si me demore, si me funciono en la anterior versión (la versión 0.90.1723) pero en la versión actual (la 1.0.2164) no me carga (perdón si vuelvo a repetir la pregunta)
